build_tools/benchmarks/reporting/data/a100_baseline.csv

```diff
@@ -1,106 +1,106 @@
 model,engine,dialect,device,shape_type,data_type,iter/sec,ms/iter,vs. PyTorch/TF,iterations,param_count,tags,notes,datetime
-albert-base-v2,tf,mhlo,cuda,static,int32,9.16034128774362,109.16623830795288,=,100,11M,nlp;bert-variant;transformer-encoder,12 layers; 128 embedding dim; 768 hidden dim; 12 attention heads; Smaller than BERTbase (11M params vs 109M params); Uses weight sharing to reduce # params but computational cost is similar to BERT.,2022-11-06 10:37:03.949918
-albert-base-v2,shark_python,mhlo,cuda,static,int32,55.95365654789094,17.871932983398438,6.11x faster,100,,,,2022-11-06 10:37:05.842975
-albert-base-v2,shark_iree_c,mhlo,cuda,static,int32,124.68827930174565,8.02,13.61x faster,100,,,,2022-11-06 10:37:07.114511

```

Comment:
This (baseline data for benchmarks of multiple frameworks) is growing a bit beyond what I'm comfortable keeping in the core IREE repository. We should consider moving some of this into a new repository IMO: https://github.com/iree-org/iree/issues/10244

Each change like this grows the size of the git repository, and data files like these could even be stored in cloud storage somewhere.

Reply:
Good point. I moved the csv files to the cloud and updated the workflow to download the baselines.